docs/great-houses/house-of-daos.mdx

```diff
@@ -1,104 +1,119 @@
-
-### Forums
-
-[DAOtalk](https://daotalk.org) - A forum for everything-DAOs.
-[MetaSquare](https://forum.metagame.wtf/) - MetaGame’s forum, all topics related to decentralized governance welcome.
-[MetaCartel Forums](https://forum.metacartel.org/) - MetaCartel’s forums.
-
-### Events
-
-[DAO Rush Week](https://twitter.com/daorushweek) - Happening once a quarter, DAO Rush Week is a must-attend event for anyone interested in DAO. All the DAOs from the Ethereum space and wider are summoned to present their missions, host parties & initiate new members.
-
-### Friendly DAOs
-
-#### Builder DAOs
-
-[Deep Work](https://deepwork.studio/) - Deep Work is a distributed product design studio that helps highly technical teams move to product market fit, faster.
-[Raid Guild](https://raidguild.org/) - A Decentralized Collective of Mercenaries Ready to Slay Your Web3 Product Demons - also slaying their own demons.
-[1Hive](https://1hive.org/) - Exploring how to create and organize a global community and using magic internet money as a catalyst.
-[dOrg](https://dorg.tech/) - We know how important it is to attract users, developer talent, and investors. We can design, code and ship your blockchain product in no time.
-[MetaGammaDelta](https://twitter.com/metagammadelta) - An inclusive and empowering society supporting women-led projects.
-[MetaFactory](https://t.co/27fiwh7Oym?amp=1) - A DAO ran platform for the creation of community-owned brands and custom digi-physical merchandise.
-[Token Engineering Community](https://tokenengineeringcommunity.github.io/website/) - Here to answer the following question: "What does it take to do token engineering, and how can we forward this nascent discipline together?"
-[LexDAO](https://lexdao.org/) - The decentralized legal engineering guild.
-[MetaFam](https://mainnet.aragon.org/#/metafam) - A decentralized collective building the infrastructure for MetaGame and helping people level up their game.
-YourDAO - This could be you. Do you already have a DAO? Want to start one?
-
-#### Funding DAOs
-
-[MetaCartel](https://www.metacartel.org/) - An ecosystem of builders giving grants to decentralized applications, scaling solutions & DAOs on the Ethereum blockchain.
-[Moloch DAO](https://www.molochdao.com/) - Awards grants to advance the Ethereum ecosystem.
-[MetaCartel Ventures](https://metacartel.xyz/) - A for-profit, venture capital DAO.
-[The LAO](https://www.thelao.io/) - A global group of Ethereum enthusiasts and experts supporting the work of Ethereum builders.
-
-#### DAO Funding
-
-Gitcoin - Another route you could take is playing Gitcoin’s quadratic matching game.
-Panvala - Panvala is the sustainable treasury for communities to share.
+<details>
 
-#### Moar on DAOs
+<summary>🤨***Wait, WTF are DAOs???***</summary>
 
-[DeepDAO.io](http://deepdao.io) - All Ethereum-based DAOs.
-[DAO Cultists & Cuntrarians](https://twitter.com/i/lists/1282667877984800768) - Our Twitter list, a healthy mix.
+<br>
```

Comment:
I think this may cause validation issues at build time. `<br>` should be `<br/>`. :smile: